docs/getting-started.md

```diff
@@ -197,8 +197,9 @@ sudo ip link set dev "$TAP_DEV" up
 
 # Enable ip forwarding
 sudo sh -c "echo 1 > /proc/sys/net/ipv4/ip_forward"
+sudo iptables -P FORWARD ACCEPT
```

Comment:
A lot of the iptable FORWARD rules are redundant with this.

Reply:
Ahh, I now see the other parts of your diff. Yes, everything still works without them :D